api/src/main/java/io/opentelemetry/metrics/Meter.java

```diff
@@ -127,7 +128,8 @@
   /**
    * Returns a builder for a {@link GaugeDouble}.
    *
-   * @param name the name of the metric.
+   * @param name the name of the metric. Should be a ASCII string with a length no greater than 255
+   *     characters.
    * @return a {@code GaugeDouble.Builder}.
    * @throws NullPointerException if {@code name} is null.
    * @throws IllegalArgumentException if different metric with the same name already registered.
```

Comment:
Nit: update this with illegal metric names.

Reply:
Done.